lib/App/Config/Chronicle.pm

```diff
@@ -110,6 +110,29 @@ has setting_name => (
     default  => 'settings1',
 );
 
+=head2 refresh_interval
+
+How much time (in seconds) should pass between C<check_for_update> invocations until
+it actually will do (a bit heavy) lookup for settings in redis.
+
+Default value is 10 seconds
+
+=cut
+
+has refresh_interval => (
```

Comment:
Validation has been added. 

The possible key collisions should be cared separately, may be with complete class re-design